File: core/src/org/rexellentgames/dungeon/debug/ShadowCommand.java
package org.rexellentgames.dungeon.debug;

import org.rexellentgames.dungeon.entity.level.Level;

public class ShadowCommand extends ConsoleCommand {
	{
		shortName = "/sd";
		name = "/shadow";
		description = "Toggles shadows";
	}

	@Override
	public void run(Console console, String[] args) {
		super.run(console, args);

		Level.SHADOWS = !Level.SHADOWS;
	}
}

File: core/src/org/rexellentgames/dungeon/debug/ShadowCommand.kt
package org.rexellentgames.dungeon.debug

import org.rexellentgames.dungeon.entity.level.Level

class ShadowCommand : ConsoleCommand("/shadow", "/sd", "Toggles shadows") {
    override fun run(console: Console, args: Array<String>) {
        super.run(console, args)

        Level.SHADOWS = !Level.SHADOWS
    }
}